app-games/src/main/java/cm/aptoide/pt/app_games/MainActivity.kt

```diff
@@ -1,20 +1,65 @@
 package cm.aptoide.pt.app_games
 
+import android.content.Intent
 import android.os.Bundle
 import androidx.activity.compose.setContent
+import androidx.activity.result.contract.ActivityResultContracts
 import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.runtime.LaunchedEffect
 import androidx.navigation.compose.rememberNavController
 import cm.aptoide.pt.app_games.home.MainView
+import cm.aptoide.pt.app_games.installer.notifications.InstallerNotificationsBuilder
+import cm.aptoide.pt.app_games.launch.AppLaunchPreferencesManager
+import cm.aptoide.pt.install_manager.InstallManager
 import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import javax.inject.Inject
 
 @AndroidEntryPoint
 class MainActivity : AppCompatActivity() {
+
+  @Inject
+  lateinit var appLaunchPreferencesManager: AppLaunchPreferencesManager
+
+  @Inject
+  lateinit var installManager: InstallManager
+
+  val requestPermissionLauncher =
+    registerForActivityResult(ActivityResultContracts.RequestPermission()) { }
+
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
+    CoroutineScope(Dispatchers.IO).launch {
```

Comment:
This first launch logic does not do anything. It can be removed from here and from the shared preferences